.teamcity/src/main/kotlin/configurations/DocsTest.kt

```diff
@@ -12,14 +12,18 @@ import model.TestSplitType.EXCLUDE
 import model.TestSplitType.INCLUDE
 import model.prepareTestClassesStep
 
+fun asDocsTestId(model: CIBuildModel, os: Os): String {
+    return "${model.projectId}_DocsTest_${os.asName()}"
```

Comment:
🤔 I thought that maybe it doesn't make sense to keep the os/Java version in the ID, similarly to what we do with the functional test projects, so we keep the history when we change Java versions. WDYT?

Reply:
> it doesn't make sense to keep the os/Java version in the ID

You mean " it doesn't make sense to keep the Java version in the ID", right? Because os is necessary. I realized this and removed java version already.

Reply:
I thought maybe we should come up with artificial IDs like for the functional tests. Though I suppose the OS should be fine for now.